### DISCUSSION TITLE
How to generate 1/4 Cylinder and apply symmetry boundary conditions

### DISCUSSION THREAD

**Discussion by RWTHLHK** [idaholab/moose#28015] [Q&A Meshing]
```
[Mesh]
  [base]
    type = AnnularMeshGenerator
    dmax = 90
    nr = 4
    nt = 5
    rmin = 0
    rmax = 4
  []
  [gauge]
    type = MeshExtruderGenerator
    extrusion_vector = '0 0 10'
    bottom_sideset = 'bottom'
    top_sideset = 'top'
    num_layers = 10
    input = base
  []
  [./left]
    type = BoundingBoxNodeSetGenerator
    new_boundary = left
    bottom_left = '0 0 0'
    top_right = '0 -4 10'
    input = gauge
  [../]
[]
```
In order to generate 1/4 Cylinder and apply symmetry boundary conditions, I must add 2 new boundaries surface left and back. But how can the boundary "left" created by this mesh be a straight line instead of a surface? (I also tried top_right(0,4,10), it is still a line, then I am totally confused).

**Comment by RWTHLHK** [comment]
Peacock always leads to segment fault, so I used Paraview to see the mesh.

**Comment by GiudGiud** [comment]
Hello

Use this generator instead for extruding
https://mooseframework.inl.gov/source/meshgenerators/AdvancedExtruderGenerator.html

It will turn the 1D lines sidesets in the 2D mesh into 2D surface sidesets in the 3D mesh

Otherwise you can easily regenerate such surfaces using a ParsedGenerateSideset
